examples/stocks/test_driver/scroll_perf.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:stocks/main.dart' as app;
+
+void main() {
```

Comment:
Is this where we want to put this file?  I wonder if we should have it be in `examples/stocks/test/driver/scroll_perf.dart`.  We've been trying to have Flutter apps follow the Dart conventions for packages.